api/cmd/cache_verify.go

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/gjson"
+
+	"github.com/apisix/manager-api/internal/conf"
+	"github.com/apisix/manager-api/internal/handler/cache_verify"
+	"github.com/apisix/manager-api/internal/log"
+)
+
+var (
+	port                     int
+	host, username, password string
+)
+
+type response struct {
+	Data cache_verify.OutputResult `json:"data"`
+}
+
+func newCacheVerifyCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "cache-verify",
+		Short: "verify that data in cache are consistent with that in ETCD",
+		Run: func(cmd *cobra.Command, args []string) {
+			conf.InitConf()
+			log.InitLogger()
+
+			port = conf.ServerPort
+			host = conf.ServerHost
+			if host == "0.0.0.0" || host == "" {
+				host = "127.0.0.1"
+			}
+			username = conf.AuthConf.Users[0].Username
+			password = conf.UserList[username].Password
+			token, err := getToken()
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "get token failed: %s\n", err)
+				return
+			}
+
+			url := fmt.Sprintf("http://%s:%d/apisix/admin/cache_verify", host, port)
```

Comment:
Don't support https ?

Reply:
@starsz 
do you mean that it's possible to use only https
and the http* url will become unavailable?
What do you think it's the right way here?

Reply:
Since the manager-api doesn't support only use https now.
So I think I will resolve this conversation.